src/api/objects/ObjectAPI.js

```diff
@@ -781,6 +781,28 @@ export default class ObjectAPI {
             && domainObject.location !== this.makeKeyString(objectPath[1].identifier);
     }
 
+    /**
+     * Check if a child domainObject is an alias (link)
+     * by comparing the it's location to the parents identifier.
+     * If the location does not match the parent's identifier, then
+     * the domainObject is an alias.
+     *
+     * @param {DomainObject} childDomainObject
+     * @param {DomainObject} parentDomainObject
+     * @returns {boolean} true if alias, false if not
+     */
+    isAlias(childDomainObject, parentDomainObject) {
```

Comment:
Let's name this `isLink` since that is the nomenclature we've historically always used.

Reply:
Thanks for making this an API function. I've no idea how we've made it this far without there being an API function for this 😄

Reply:
No problem! Yeah, `isLink` makes more sense!

Reply:
Oh wait, I just discovered `isObjectPathToLink` which is doing the same thing, but has a weird function signature. I don't know why it requires you to provide both an object _and_ an objectPath. Let's use the existing function and file a followup to refactor that function so it only requires an objectPath.